README.md

```diff
@@ -4,7 +4,7 @@
 # Visual Studio Code extension for Ansible
 
 ## Overview
-This extension is to help user use [ansible](https://www.ansible.com/) in VSCode.
+This extension is to help user use [Ansible](https://www.ansible.com/) in VSCode.
 
```

Comment:
I think "user use" doesn't sound well. 
Also "This extension is to help" -> "This extension helps ..."

Or maybe it should be sth like: "This extension makes it easy to edit and execute Ansible playbooks from within Visual Studio Code"? just thinking

Reply:
nice, fixed